### DISCUSSION TITLE
Levels used in tips

### DISCUSSION THREAD

**Discussion by danconde** [azadbolour/eagerwords-community#9] [Q&A] [answered]
In the web version, if you use Get Tip, at which level does the program provide a tip.  At the level you are playing at, at the bot's level?  In other words, using a tip does not provide for an equal game against a bot?

It is stated as 'Yes. You can ask for a tip. The tip request causes the system to find a word to play for you (if possible) and to move its first letter to the board. Subsequent tips for the same play will move successive letters of the word to the board. Generally, after one or two tips you will be able to discern the rest of the word to play.

Be aware that tip moves do not contribute to your score, and there are no **guarantees** as to the quality of the tips.'

**Comment by azadbolour** [comment, marked as answer]
It uses the intermediate level for all games. Here is the faq paragraph:

"
EagerWords uses its bot at the intermediate level to find tips. The
intermediate level
    has a vocabulary of only 10000 words. It represents a compromise
between the need to
    provide useful tips, and the need to discourage tip play except as a
last resort."

**Comment by danconde** [comment]
Sorry that I did not read what was already documented!
